src/functions/base_function_handler_type_test.ts

```diff
@@ -103,6 +103,32 @@ Deno.test("BaseSlackFunctionHandler with any inputs and any outputs", () => {
   assertEquals(result.outputs?.out, inputs.in);
 });
 
+Deno.test("BaseSlackFunctionHandler with no inputs and error output", () => {
+  // deno-lint-ignore no-explicit-any
+  const handler: BaseSlackFunctionHandler<any, any> = () => {
+    return {
+      error: "error",
+    };
+  };
+  const { createContext } = SlackFunctionTester("test");
+  const inputs = { in: "test" };
+  const result = handler(createContext({ inputs }));
```

Comment:
The test title says that this doesn't have any `inputs`, so we can set it to an empty object here like `const result = handler(createContext({ inputs: {} }));`